test/test-daemon.c

```diff
@@ -100,6 +100,97 @@ flush_stream_to_string (GInputStream *stream)
 }
 
 static void
+test_feature_testing_works (DaemonFixture *fixture,
+                            gconstpointer user_data)
+{
+  SoupSession *session;
+  SoupRequestHTTP *req;
+  GInputStream *stream;
+  GError *error = NULL;
+  GString *reply;
+  SoupMessage *message;
+  gchar *req_uri;
+
+  // Test for a known feature.
```

Comment:
OK, but again that doesn't seem to be obviously documented anywhere (and they've been valid in C since 1999).